.github/workflows/dapr-release-notes.yml

```diff
@@ -0,0 +1,39 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-release-notes
+
+on:
+  repository_dispatch:
+    types: release_notes
+jobs:
+  build:
+    name: Generate release notes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: pip install PyGithub
+      - name: Generate release notes
+        env:
+          GITHUB_TOKEN: ${{ secrets.DAPR_BOT_TOKEN }}
+        run: python ./.github/scripts/generate_release_notes.py
+      - name: Commit and push to repository
+        env:
+          GITHUB_TOKEN: ${{ secrets.DAPR_BOT_TOKEN }}
+          COMMIT_MSG: |
+            Generating Dapr release notes.
+            skip-checks: true
+        run: |
+          git config user.email "Dapr Bot"
+          git config user.name "daprweb@microsoft.com"
+          # Update origin with token
+          git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+          # Checkout the branch so we can push back to it
+          git checkout master
```

Comment:
`RELEASE_VERSION = ${{ github.event.inputs.releaseVersion }}` required input.

This needs to change to the release-notes-${RELEASE_VERSION} branch preferably. 

Create a PR from release-notes-${RELEASE_VERSION} branch to release-${RELEASE_VERSION} branch? 

Install gh : https://github.com/cli/cli/blob/trunk/docs/install_linux.md#debian-ubuntu-linux-apt
from manual
`gh will respect tokens set using GITHUB_TOKEN.`

``` 
gh pr create --base release-${RELEASE_VERSION}  \
    --title "Release Notes for ${RELEASE_VERSION}" --body ''
```

Reply:
I agree to the PR approach.

Reply:
I did it in a different way. See sample PR: https://github.com/artursouza/dapr/pull/5